### PR TITLE
Global fixes

### PR DESCRIPTION
**Pull request by hibbert**
## what

Currently for a secondary cluster with this module you get the following error:

```
Error: error creating RDS cluster: InvalidParameterCombination: Cannot specify user name for cross region replication cluster
```

This cannot be worked around as `admin_user` has a default of "admin", this sets the default to an empty string 

## why
* So can have a global cluster

## references
* N/A

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -58,14 +58,14 @@ variable "db_port" {
 
 variable "admin_user" {
   type        = string
-  default     = "admin"
-  description = "(Required unless a snapshot_identifier is provided) Username for the master DB user"
+  default     = ""
```
Comment (Nuru) [inline variables.tf:61]:
```suggestion
  default     = "admin"
```
To preserve backwards compatibility, we do not want to change default values.

**Review by Nuru — CHANGES_REQUESTED** [review]

**Comment by hibbert** [comment]
No worries, was able to set `admin_user=""`

**Pull request closed without merging**